README.md

```diff
@@ -1,5 +1,7 @@
 # Azure IoT Edge Dev Tool
 
+[![Build Status](https://travis-ci.org/Azure/iotedgedev.svg?branch=master)](https://travis-ci.org/Azure/iotedgedev)
```

Comment:
This is just to have the fancy badge on the README if we do indeed use Travis. =)